src/PostgREST/Request/Types.hs

```diff
@@ -220,3 +222,11 @@ type SingleVal = Text
 
 -- | Represents a list value in a filter, e.g. id=in.(val1,val2,val3)
 type ListVal = [Text]
+
+-- | Three-valued logic values
+data TrileanVal
+  = TriTrue
+  | TriFalse
+  | TriNull
+  | TriUnknown
+  deriving Eq
```

Comment:
```suggestion
```

Reply:
It gives an error if removed:

```
src/PostgREST/Request/Types.hs:215:13: error:
    • No instance for (Eq TrileanVal)
        arising from the first field of ‘Is’ (type ‘TrileanVal’)
      Possible fix:
        use a standalone 'deriving instance' declaration,
          so you can specify the instance context yourself
    • When deriving the instance for (Eq Operation)
    |
215 |   deriving (Eq)
```

Reply:
Ah, I was mislead by the coverage report saying it was unused. I remember reading something about coverage in that case and how the report can be misleading. Might be able to find it later.